robottelo/common/constants.py

```diff
@@ -203,8 +203,8 @@
 
 PUPPET_MODULE_NTP_PUPPETLABS = "puppetlabs-ntp-3.2.1.tar.gz"
 
-# All satellite permissions
-# API test_permission_v2 ensures that this will not get obsolete
+#: All permissions exposed by the server.
+#: :mod:`tests.foreman.api.test_permission` makes use of this.
```

Comment:
You can also document class-wide variables with that same syntax. The variables being documented need to be public, though. This will not work:

    #: My documentation.
    _MY_PRIVATE_VARIABLE